frontend/apps/migration-web/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "migration-web",
+  "private": true,
+  "version": "0.1.0",
+  "dependencies": {
+    "@langchain/core": "0.3.42",
+    "@langchain/openai": "0.4.4",
+    "langchain": "0.3.19",
+    "next": "15.1.2",
+    "react": "18.3.1",
+    "react-dom": "18"
+  },
+  "devDependencies": {
+    "@biomejs/biome": "1.9.3",
+    "@types/node": "22.9.0",
+    "@types/react": "18",
+    "@types/react-dom": "18",
+    "typed-css-modules": "0.9.1",
+    "typescript": "5"
+  },
+  "scripts": {
+    "build": "pnpm gen && next build",
+    "dev": "pnpm run '/^dev:.*/'",
+    "dev:css": "tcm components --watch",
+    "dev:next": "next dev --port 3003",
```

Comment:
👍 
https://github.com/liam-hq/liam/blob/42476a58ec4898e512db5f184f2e159b9a66f1c7/CONTRIBUTING.md?plain=1#L35-L39 

- Port 3003 is not in use.  
- No need to include details about the migration web here.